### ISSUE TITLE
Add a "Transparent" Option to the Color Picker

### ISSUE THREAD

**Issue by gyanl** [prikhi/pencil#695]
Current method to remove stroke from a shape : 
Select shape
Click on stroke option
Reduce opacity to 0 by clicking 10 times, as the slider reduces by 10% only.

Suggested solutions:
1. Add a 'No Stroke' option either as a checkbox or as a color option shown by a box with a red cross as in Adobe Photoshop & Illustrator
2. Add an option to input the opacity into a textbox
3. Improve the slider so it can be made zero more quickly/smoothly.

![pencil](https://cloud.githubusercontent.com/assets/9626697/7495857/21427c30-f42e-11e4-8998-3795e4239592.png)

**Comment by prikhi** [comment]
There are 2 ways to do this:
1. In the `line` section to the right of the `color` toolbar section, set the number input to `0`(this controls the border width).
2. Right click the shape & choose `Actions -> Remove Border`.

Does this seem OK?

**Comment by gyanl** [comment]
Yeah that actually seems quite reasonable, just not what I'm used to.

It might still be a good idea to add a transparent color option in the
fill, stroke and line options, even if all that does is change transparency
to 0.
